openprescribing/frontend/management/commands/import_measures.py

```diff
@@ -94,6 +94,11 @@ def add_arguments(self, parser):
     def parse_options(self, options):
         """Parse command line options
         """
+        if len([
+                x for x in (options['start_date'], options['end_date'])
+                if x is not None]) == 1:
+            raise CommandError(
+                '--start_date and --end_date must be given together')
```

Comment:
What about:

```
if bool(options['start_date']) != bool(options['end_date']):
    raise CommandError(...)
```

Reply:
Or even, for more clarity:

```
if options['start_date']:
    if not options['end_date']:
        raise ...

if options['end_date']:
    if not options['start_date']:
        raise ...
```

Reply:
I prefer your first suggestion as it's clearer than what I wrote but still doesn't involve repetition. The CommandError itself clarifies the intention sufficiently, IMO